docs/appendix.rst

```diff
@@ -594,6 +604,58 @@ the SSL module of Python which hands it up to pywbem. The error indicates that
 OpenSSL on the client side cannot deal with the cipher used by the server
 side. This was fixed by upgrading OpenSSL on the client OS to version 1.1.1.
 
+.. _`Install fails, Externally-managed-environment error`:
+
+Install fails, Externally-managed-environment error
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. index:: pair: troubleshooting; Externally-managed-environment error
+.. index:: pair: installation failure; Externally-managed-environment error
+
+This error is caused by the OS distribution adopting the changes defined by
+`Python PEP 668`_ (Marking Python base environments as “externally managed”) which
+sets configuration information so that pip (version 23.0+) will only install packages that are
+not part of the OS distibution into virtual environments and will not install
+them into any of the system-wide directories. This forces the separation of user
+packages from OS distribution installed packages.
+
+On newer versions of some operating systems (Ex. Ubuntu 23.04, Debian 12,
+etc.) installed from the OS distribution, you may get an error message such as:
+
+.. code-block:: text
+
+        error: externally-managed-environment
+
+    × This environment is externally managed . . .
+    . . .
+
+The best solution to this issue is to ``always`` install pywbem into a virtual
+environment as recommended in the pywbem installation.
+
+There are alernatives to allow installation of pywbem system-wide if required
```

Comment:
I suggest to use the term "into the system Python" instead of "system-wide".

Reply:
changed to system Python directories